### PR TITLE
ScaleUP PDS Dataservices Test

### PR DESCRIPTION
**Pull request by Madanagopal19**
ScaleUPDataServices Test scales up the data services nodes and validates the health state of the deployment post upgrade

ENV:
export UPDATE_REPLICAS="supported replicas count"

Test Execution logs - https://gist.github.com/Madanagopal19/a1730bdd3282d1335fa315247232555a

### CODE REVIEW

**Comment by bhsrinivasan-px** [comment]
LGTM

**Review by bhsrinivasan-px — APPROVED** [review]
LGTM

File: tests/pds/dataservice_test.go [hunk 1]
```diff
@@ -130,12 +129,182 @@ var _ = BeforeSuite(func() {
...
+			oldDSVersion = pdslib.GetAndExpectStringEnvVar(VersionToBeUpdated)
+		}
+		updatedRep, err := pdslib.GetAndExpectIntEnvVar(envUpdateReplicas)
+		Expect(err).NotTo(HaveOccurred())
+		updatedrep = int32(updatedRep)
+
+	})
+
+	It("scale up dataservice nodes", func() {
+		Step("Deploy Data Services", func() {
```
Comment (lsrinivas-pure) [inline tests/pds/dataservice_test.go:167]: Are we deploying dataservice even when DeployAllDataService is false?

File: deployments/deploy-ssh.sh [hunk 2]
```diff
@@ -563,8 +563,14 @@ spec:
       value: "${CLUSTER_TYPE}"
     - name: TARGET_KUBECONFIG
       value: "${TARGET_KUBECONFIG}"
-    - name: TARGET_CLUSTER_NAME
-      value: "${TARGET_CLUSTER_NAME}"
+    - name: IMAGE_TO_UPDATE
```
Comment (lsrinivas-pure) [inline deployments/deploy-ssh.sh:566]: what is the image here? and where is this being used?

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]
can we have sync up on this PR?

File: tests/pds/dataservice_test.go (hunk 1 above)
Comment (Madanagopal19) [reply tests/pds/dataservice_test.go:167]: Yes, we are deploying data service if DeployAllDataService is set to false.
we have two env params  DeployAllDataService and DataService.
If DeployAllDataService flag is required to run `DeployAllDataServices` test, For other test we need only DataService flag to be set

**Comment by Madanagopal19** [comment]
> can we have sync up on this PR?



> can we have sync up on this PR?

sure @lsrinivas-pure .

**Comment by Madanagopal19** [comment]
closing this PR because of conflicts

**Pull request closed without merging**